cob_twist_controller/include/cob_twist_controller/cob_twist_controller_debug.h

```diff
@@ -95,14 +95,14 @@ class CobTwistController
 	
```

Comment:
once the new structure is verified and tested, we also might want to drop the *_debug code